src/debug.h

```diff
@@ -53,6 +53,12 @@
 #define __FUNCTION_NAME__ __func__
 #endif
 
+#define fatal(cond) \
+    if( !( cond ) ) { \
+        debugmsg( "Assertion failed: %s", STRING( cond ) ); \
+        std::terminate(); \
```

Comment:
That's a known issue with `debugmsg` that has been noticed before and should be fixed upstream. I'll probably tackle this in a further PR once I dig out the windows API reference